modules/nodes-cluster-enabling-features-cluster.adoc

```diff
@@ -40,9 +29,29 @@ kind: FeatureGate
 metadata:
   name: cluster
 spec:
-  featureSet: "TechPreviewNoUpgrade" <1>
+  featureSet: CustomNoUpgrade,TechPreviewNoUpgrade,LatencySensitive,IPv6DualStackNoUpgrade <1>
+  customNoUpgrade: <2>
+  disabled:
+    - ExpandCSIVolumes
+    - ExpandInUsePersistentVolumes
+  enabled:
+    - VolumePVCDataSource
+    - VolumeSnapshotDataSource
+    - BlockVolume
+    - CSIBlockVolume
 ----
-<1> Add the `featureSet: "TechPreviewNoUpgrade"` parameter.
+<1> Add the `featureSet: parameter and the feature gates you want to activate.
+<2> For the `CustomNoUpgrade` feature set, specify the features you want to 
+enable or disable.
```

Comment:
@sttts Can the user get lists of features that the TechPreviewNoUpgrade and CustomNoUpgrade affect? 

We have this unused topic that lists features that can be turned on/off. But this seems unmanageable from version to version

Reply:
I don't think users can get a list.

`oc explain ...` could output such a list if the API docs have it. Today this is not the case.

cc @danwinship @deads2k @rphillips, who conceived this API afaik.

Reply:
I wasn't involved with the design of the API. I just added the `IPv6DualStackNoUpgrade` feature set.

FTR as far as I could tell, the "prevents upgrade" stuff in the API docs is a lie. I couldn't find anything anywhere that actually prevents upgrades if you are using a "NoUpgrade" featureset